content/docs/pipelines/installation/standalone-deployment.md

```diff
@@ -219,9 +220,29 @@ Run:
 kubectl create clusterrolebinding your-binding --clusterrole=cluster-admin --user=[your-user-name]
 ```
 
-### Pipeline samples that require "user-gcp-sa" secret
+### Securing the cluster with fine-grained GCP permission control
 
-If sample pipeline requires a "user-gcp-sa" secret, you could create one by:
+#### Workload Identity
+
+> Workload identity is the recommended way for your GKE applications to consume services provided by Google APIs. You accomplish this by configuring a Kubernetes service account to act as a Google service account. Any Pods running as the Kubernetes service account then use the Google service account to authenticate to cloud services.
+
+Referenced from [workload identity documentation](https://cloud.google.com/kubernetes-engine/docs/how-to/workload-identity). Please read this doc for detailed introduction.
+
+To bind workload identities for kubernetes service accounts (KSAs) provided by pipelines standalone deployment, we provide some helper scripts to set up workload identity bindings for pipelines workloads.
+[This user-friendly bash script](https://github.com/kubeflow/pipelines/blob/master/manifests/kustomize/gcp-workload-identity-setup.sh) help you create GCP service accounts and bind them to kubernetes service accounts used by pipelines workloads.
```

Comment:
Tech writer hint: Remove "user-friendly", as it doesn't add any meaning to the content, and people may become upset if they try using the script and discover that it's not as easy as it may seem.

Reply:
Thanks! That sounds reasonable. The distinction I wanted to clarify is that `workload-identity-setup.sh` script is meant for interactive command line usage with enough messages.
`

`wi-utils.sh` is bare minimal script that is easy to read and use programmatically.

I think it's important to introduce the difference here, do you have any suggestion on a better wording to convey this message?

Reply:
I see! I think the wording you gave in this comment is very good - it explains to the reader how the scripts differ in terms of usage.

How about something like this:

```
The following helper scripts bind workload identities for the Kubernetes service
accounts (KSAs) provided by the Kubeflow Pipelines Standalone deployment, to set up workload 
identity bindings for pipelines workloads:

* [`gcp-workload-identity-setup.sh`](https://github.com/kubeflow/pipelines/blob/master/manifests/kustomize/gcp-workload-identity-setup.sh) helps you create GCP service
 accounts and bind them to the Kubernetes service accounts used by pipeline
 workloads. This script provides an interactive command line dialog with
 appropriate messages

 * [`wi-utils.sh`](https://github.com/kubeflow/pipelines/blob/master/manifests/kustomize/wi-utils.sh) provides utility bash functions with minimal interaction,
 which you can use to customize your setup. The minimal interactivity of this
 script makes it easy to read and use programmatically.
 ```

Reply:
That sounds great! Thanks a lot. I will change

Reply:
Changed